modules/core/src/test/scala/io/chrisdavenport/fuuid/FUUIDArbitraries.scala

```diff
@@ -0,0 +1,22 @@
+package io.chrisdavenport.fuuid
+
+import org.scalacheck._
+import java.util.UUID
+
+trait FUUIDArbitraries {
+  private implicit val arbUuid: Arbitrary[UUID] = Arbitrary( for {
```

Comment:
There is an implementation for `Gen[UUID]` in Scalacheck [here](https://github.com/rickynils/scalacheck/blob/master/src/main/scala/org/scalacheck/Gen.scala#L831-L837). Why do you define your own generator? Just curious about it :thinking:

Reply:
Because you can't use things when you are not aware they exist. Updating now.